### ISSUE TITLE
Issue with trailing whitespace in translation string when used with a directive

### ISSUE THREAD

**Issue by kyris9** [ngx-translate/core#790]
**I'm submitting a ...**
```
[x] bug report => check the FAQ and search github for a similar issue or PR before submitting
[ ] support request => check the FAQ and search github for a similar issue before submitting
[ ] feature request
```

**Current behavior**
When the value of a key in the JSON file ends with a white-space, the language change does not have an effect **when used with a directive**. This works correctly when used with a pipe or with the service.

**Expected/desired behavior**
The value should get translated as defined in the JSON files.

**Reproduction of the problem**
**html**
`<span id="lblSearchCriteria" translate>common.terms.searchCriteriaLine</span>`
**en.json**
`{ "common": { "terms": { "searchCriteriaLine": "Search Criteria - " } } }`
**el.json**
{ "common": { "terms": { "searchCriteriaLine": "Κριτήρια Αναζήτησης - " } } }`
`

**Please tell us about your environment:**

* **ngx-translate version:** 9.1.1

* **Angular version:** 5.1.1
 
* **Browser:** [Chrome 64.0.3282.140]

**Comment by CodeAndWeb** [comment]
Confirmed

https://stackblitz.com/edit/github-ad9uqm

**Comment by liviu-c** [comment]
It seems this issue with the spaces (at the begining or at the end) is also available in the last build v11.0.0

https://stackblitz.com/edit/github-ad9uqm-vqauiz

**Comment by liviu-c** [comment]
any news on this? the issue is valid in 11.0.1 as well.

**Comment by Karl-Fokum-LogMeIn** [comment]
Any news on this issue?

**Comment by Bat-Orshikh** [comment]
is there any on this issue?

**Comment by jloupdef** [comment]
Not exactly the same problem as mine but might be resolved by my pending pull request #1117 as stated error is :
sh: 0: Can't open /etc/init.d/xvfb
The command "sh -e /etc/init.d/xvfb start" failed and exited with 127 during .
